use-cases/NVIDIA-GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,296 @@
+# Using Nvidia GPU device with Kata Containers
+
+- [Using Nvidia GPU device with Kata Containers](#using-nvidia-gpu-device-with-kata-containers)
+	- [Hardware Requirements](#hardware-requirements)
+	- [Host BIOS Requirements](#host-bios-requirements)
+	- [Host Kernel Requirements](#host-kernel-requirements)
+	- [Install and configure Kata Containers](#install-and-configure-kata-containers)
+	- [Build Kata Containers kernel with GPU support](#build-kata-containers-kernel-with-gpu-support)
+	- [NVIDIA GPU Pass-Through mode with Kata Containers](#nvidia-gpu-pass-through-mode-with-kata-containers)
+	- [NVIDIA vGPU mode with Kata Containers](#nvidia-vgpu-mode-with-kata-containers)
+	- [Install Nvidia Driver in Kata Containers](#install-nvidia-driver-in-kata-containers)
+
+
+A Nvidia GPU device can be passed to a Kata Containers container using GPU passthrough (NVIDIA GPU Pass-Through mode) as well as GPU mediated passthrough (NVIDIA vGPU mode). 
+
+NVIDIA GPU Pass-Through mode (one VM to one physical GPU)  is one flavor of graphics virtualization approach. This flavor allows direct assignment of an entire GPU to a single user, passing the native driver capabilities through the hypervisor without any limitations.
+
+NVIDIA vGPU mode enables multiple virtual machines (VMs) to have simultaneous, direct access to a single physical GPU.
+
+| Technology | Description | Behaviour | Detail |
+| --- | --- | --- | --- |
+| NVIDIA GPU Pass-Through mode | GPU passthrough | Physical GPU assigned to a single VM | Direct GPU assignment to VM without limitation |
+| NVIDIA vGPU mode | GPU sharing | Physical GPU shared by multiple VMs | Mediated passthrough |
+
+## Hardware Requirements
+NVIDIA GPUs Recommended for Virtualization:
+
+- NVIDIA Tesla (T4, M10, P6, V100...)
+- NVIDIA Quadro RTX 6000/8000
+
+About large BARs device (MMIO space above 4G): 
```

Comment:
Maybe make the BARS section a subsection of Note, with a little sentence explaining that some hardware requires a larger window (iiuc)

Reply:
I had moved it to the `Host BIOS Requirements` section